src/index.js

```diff
@@ -16,10 +16,22 @@ export default function App () {
           </Route>
         </Routes>
       </BrowserRouter>
-   <Services />
-    <Footer />
-  </div>
-
+      <Services />
+      <ServiceArea />
+        <div className='flex flex-col p-16 bg-green-700'>
+        <h1 className='text-center text-white'>
```

Comment:
issue: We should only have one h1 per page, and I think since the `BrowserRouter` is on this file, we'll be getting our h1's from those pages.